site/content/docs/developing/content-delivery.en.md

```diff
@@ -29,21 +29,28 @@ As of Copilot v1.20 your environment will be created with a manifest file; just
 
         ```yaml
         cdn:
-          certificate: cert
+          certificate: arn:aws:acm:us-east-1:${AWS_ACCOUNT_ID}:certificate/13245665-h74x-4ore-jdnz-avs87dl11jd
 
         http:
           certificates:
-            - cert1
-            - cert2
+            - arn:aws:acm:${AWS_REGION}:${AWS_ACCOUNT_ID}:certificate/13245665-bldz-0an1-afki-p7ll1myafd
+            - arn:aws:acm:${AWS_REGION}:${AWS_ACCOUNT_ID}:certificate/56654321-cv8f-adf3-j7gd-adf876af95
         ```
 
 ## How do I enable HTTPS traffic with CloudFront?
 
-When using HTTPS, CloudFront uses the same paradigm for certificates as specifying in the `http.certificates` field of the environment manifest for a Load Balancer. This means that there's a new field: `cdn.certificate` which is used to enable HTTPS traffic with CloudFront. However, this certificate is constrained to be imported in the region `us-east-1`.
+When using HTTPS with CloudFront, specify your certificates in the `http.certificates` field of the environment manifest, just as you would for a Load Balancer. The new field `cdn.certificate` similarly enables HTTPS traffic with CloudFront. Unlike a Load Balancer, you can only import one certificate. Because of this, we recommend that you create a new certificate in the `us-east-1` region which contains CNAME records to validate each alias that your services use in that environment.
```

Comment:
```suggestion
When using HTTPS with CloudFront, specify your certificates in the `cdn.certificate` field of the environment manifest, just as you would in the `http.certificates` field for a Load Balancer. Unlike for a Load Balancer, you can import only one certificate. Because of this, we recommend that you create a new certificate (in the `us-east-1` region) with CNAME records to validate each alias in that environment.
```